test/onnx/parse/roialign_test.cpp

```diff
@@ -41,7 +41,7 @@ TEST_CASE(roialign_test)
                           {{"coordinate_transformation_mode", "output_half_pixel"},
                            {"spatial_scale", 2.0f},
                            {"output_height", 5},
-                           {"output_width", 5},
+                           {"output_width", 3},
```

Comment:
This particular change was a big deal, btw, since the old code appeared to work fine until I gave it an output_height and output_width that were not the same.